toolchain/lower/file_context.cpp

```diff
@@ -463,4 +479,17 @@ auto FileContext::BuildType(SemIR::InstId inst_id) -> llvm::Type* {
   }
 }
 
+auto FileContext::BuildGlobalVariableDecl(SemIR::InstId inst_id)
+    -> llvm::GlobalVariable* {
+  auto var_storage = sem_ir().insts().Get(inst_id).As<SemIR::VarStorage>();
```

Comment:
Note I'm suggesting passing in the `VarStorage` above, but I'll note for future reference this could also be written as `GetAs<SemIR::VarStorage>(inst_id);`